src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomBall());
+        System.out.println(lottery.getRandomBall());
+        System.out.println(lottery.getRandomBall());
+
```

Comment:
Repeating code, maybe better use loop?

Reply:
Thank you for feedback, i commit and push new changes, but i think something wrong with my git, because new commit did not pass automatic verification. Please, check it in commits of this project. And i would be glad if you could explain if I did everything right when interacting with git

Reply:
All questions should be discussed in chat, not in the PR